build.gradle

```diff
@@ -105,25 +105,133 @@ subprojects {
     project.licenseFile = project.rootProject.file('licenses/APACHE-LICENSE-2.0.txt')
     project.noticeFile = project.rootProject.file('NOTICE.txt')
 
-    // Projects that should be formatted and checked with Spotless are
-    // listed here, by project path. Once the number of formatted projects
-    // is greater than the number of unformatted projects, this can be
-    // switched to an exclude list, and eventually removed completely.
-    def projectPathsToFormat = [
-      ':benchmarks',
-      ':build-tools',
-      ':distribution:tools:java-version-checker',
-      ':distribution:tools:keystore-cli',
-      ':distribution:tools:launchers',
-      ':distribution:tools:plugin-cli',
-      ':qa:os',
-      ':qa:wildfly',
-      ':x-pack:plugin:autoscaling',
-      ':x-pack:plugin:enrich',
-      ':x-pack:plugin:logstash'
+    // Projects that should be excluded from automatic formatting and
+    // checking with Spotless are listed here, by project path. Once the
+    // number of unformatted projects reaches zero, this mechanism will be
+    // removed.
+    //
+    // No new projects should be added here!
+    def projectPathsToExclude = [
```

Comment:
I think this list will have to be regenerated on backport.

Reply:
Yes, I was anticipating that.